Disable call to `imbue()`, which raises an exception when used with musl.  It
seems that this call is required on Windows, but not on Linux.
See https://gitlab.com/mbunkus/mkvtoolnix/-/issues/3482
--- a/src/common/locale.cpp	2023-02-13 20:39:33.365761113 -0500
+++ b/src/common/locale.cpp	2023-02-13 20:40:08.745454105 -0500
@@ -339,7 +339,9 @@
 
     try {
       std::locale::global(utf8_locale);
+#if 0
       boost::filesystem::path::imbue(utf8_locale);
+#endif
 
       boost_initialized = true;
       mxdebug_if(debug, fmt::format("initialize_std_and_boost_filesystem_locales: boost::filesystem initialized from '{0}' ({1})\n", locale_name, utf8_locale.name()));
